docs/en/writing-running-appium/tutorial/swipe/ios-mobile-element-search.md

```diff
@@ -0,0 +1,80 @@
+## iOS 'mobile:': Element search swipe
+
+To search the element on screen or scrollView use:
+- element id with 'name' argument
+- or specify NSPredicate string using 'predicateString' argument
+
+NSPredicate examples available [here](../../ios/ios-predicate.md)
```

Comment:
https://kapeli.com/cheat_sheets/NSPredicate.docset/Contents/Resources/Documents/index is a good NSPredicate cheat sheet